_content/excerpts/can-monks-practice-astrology_kotyk-j.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Can Monks Practice Astrology? Astrology and the Vinaya in China."
+authors:
+  -  "Jeffrey Kotyk"
+external_url: "https://www.academia.edu/34408763/_Can_Monks_Practice_Astrology_Astrology_and_the_Vinaya_in_China_"
```

Comment:
Let's add the book pdf as a source url: https://www.buddhismuskunde.uni-hamburg.de/pdf/4-publikationen/hamburg-buddhist-studies/hamburgup-hbs09-full.pdf